### PR TITLE
injected segment key

### PR DESCRIPTION
**Pull request by AnirudhPokala**
Segment key injected

### CODE REVIEW

**Review by nikhilbandi — CHANGES_REQUESTED** [review]
Please handle the Else case as suggested by Anup.

File: pkg/pmk/segment.go [hunk 1]
```diff
@@ -14,7 +14,10 @@ import (
 )
 
 //Added segment key for the source PRD-PMKFT Metrics-Aggregator
-const segmentWriteKey = "4jevYUNBF5sY3vZJWm5TrhfdsFzQIQ3y"
+//const segmentWriteKey = "4jevYUNBF5sY3vZJWm5TrhfdsFzQIQ3y"
+var SegmentWriteKey string
+
+//var SegmentWriteKey = os.Getenv("SEGMENT_KEY_PRD_PMKFT")
```
Comment (ParagDoke) [inline pkg/pmk/segment.go:19]: Please drop the comment.

File: pkg/pmk/segment.go [hunk 2]
```diff
@@ -34,10 +37,16 @@ func NewSegment(fqdn string, noTracking bool) Segment {
 	// mock out segment if the user wants no Tracking
 	envCheck := os.Getenv("PF9CTL_SEGMENT_EVENTS_DISABLE")
 	segmentEventDisabled, _ := strconv.ParseBool(envCheck)
+
+	// Local Case.
```
Comment (ParagDoke) [inline pkg/pmk/segment.go:40]: Comment should read "Local build case" to unambiguously capture if the build is local or run is local.

**Review by ParagDoke — COMMENTED** [review]
Must squash commits before merging. Currently there are 11 of them.

File: pkg/pmk/segment.go (hunk 1 above)
Comment (AnirudhPokala) [reply pkg/pmk/segment.go:19]: Sure dropped off.

File: pkg/pmk/segment.go (hunk 2 above)
Comment (AnirudhPokala) [reply pkg/pmk/segment.go:40]: Sure will modify.

**Review by anupbarve — APPROVED** [review]

**Review by devidask27 — APPROVED** [review]

**Review by nikhilbandi — APPROVED** [review]
Looks good.

**Pull request merged**